store/store.go

```diff
@@ -204,7 +213,7 @@ func (s *DefaultStore) loadHashFromIndex(height uint64) ([32]byte, error) {
 
 	var hash [32]byte
 	if err != nil {
-		return hash, err
+		return hash, fmt.Errorf("error loading hash by height %v: %w", height, err)
```

Comment:
```suggestion
		return hash, fmt.Errorf("error loading block hash for height %v: %w", height, err)
```

Reply:
Hey @tzdybal sorry I am a little unsure as to what you mean by the singular form in this case